src/main/webapp/WEB-INF/views/admin/vulnerability/vulnpopup.jsp

```diff
@@ -133,6 +143,53 @@
 		function unCveIdTag(cellvalue, options, rowObject){
 			return cellvalue;
 		}
+
+		/* Exports vulnerability data with xlsx format */
+		function downloadExcel() {
+			const makeRequestData = function() {
+				const searchData = $("#_vulnList").jqGrid('getGridParam','postData');
+				const parameter = {
+					ossName: searchData.ossName,
+					ossVersion: searchData.ossVersion,
+					sidx: searchData.sidx,
+					sord: searchData.sord,
+					vulnType: searchData.vulnType,
+				};
+				const type = "vulnerabilityPopup";
+
+				return {"parameter": JSON.stringify(parameter), type};
+			};
+
+			if(isMaximumRowCheck(totalRow)){
+				$('#loading_wrap_popup').show(); //Show loading layout
+
+				$.ajax({
+					type: "POST",
+					url: '<c:url value="/exceldownload/getExcelPost"/>',
+					data: JSON.stringify(makeRequestData()),
+					dataType : 'json',
+					cache : false,
+					contentType : 'application/json',
+					success: function (data) {
```

Comment:
@FRESH-TUNA Please remove space between anonymous function keyword and parenthesis.

Reply:
Thank you! i fixed it